tools.tex

```diff
@@ -1,34 +1,43 @@
 \section{Tools} \label{sec:tools}
 
-This section of the paper describes the tools used for the scrub process. With the requirement for agility, collaborative working and ease of linking to existing tools, Google Workspace  is the chosen platform onto which the scrub tools have been developed. The tool, called the Scrub Sandbox,  needs to facilitate the following:
-
+This section describes the tools used for the scrub process. 
+With the requirement for agility, collaborative working and ease of linking to existing tools, Google Workspace is the chosen platform on which all the Rubin planning tools, including the scrub tool, have been developed. 
+The particular tool that supports the annual scrub is a Google Sheets workbook called the ``Scrub Sandbox.''
+It needs to facilitate:
 \begin{enumerate}
-\item Capturing the current state.
-\item Capturing what the desired change is.
-\item Inputting flow down milestones for the upcoming FY based on higher level milestones defined by the directors office.
-\item Standardize inputs coming in from the department and teams.
-\item Easily seeing the impact of the desired change on labor and non-labor budget.
+\item capturing the current state of the operations plan for each team;
+\item capturing what the desired changes are;
+\item inputting flow down milestones for the upcoming FY based on higher level milestones defined by the directors Office;
+\item standardizing inputs coming in from the Departments and Teams;
+\item easiy visualization of the impact of the desired change on the labor and non-labor budgets;
+\item collaboration between Team Leaders as they work through their competing needs.
 \end{enumerate}
 
 \begin{figure}[h!]
 \begin{centering}
 \includegraphics[width=1.0\textwidth]{Figure3OverviewScrubSandbox}
-	\caption{ Overview of Scrub Sandbox
+	\caption{ Overview of the Scrub Sandbox tool.
 \label{fig:sandbox}}
 \end{centering}
 \end{figure}
 
-\noindent Furthermore, the tool needs to enable the following areas to be scrubbed:
+\noindent Furthermore, the tool needs to enable the following aspects of the overall operations plan to be scrubbed:
 \begin{enumerate}
-\item Work Breakdown Structure (WBS).
-\item Risks.
-\item Milestones.
-\item Labor expenditure.
+\item Work Breakdown Structure (WBS);
+\item Risks;
```

Comment:
```suggestion
\item risks;
```
if we are using semi-colons shouldn't these now be lower case because they aren't starting a new sentence?

Reply:
I wondered this too. I left them capitalized because each one refers to a named Thing in Rubin Ops (eg the Risk Register). It's a matter of taste though, and I know @ranpalgill tends to prefer lower case.